docs/production/export-and-import.md

```diff
@@ -460,11 +461,18 @@ s3_backups_bucket = # name of S3 backup bucket
 1. Run `/home/zulip/deployments/current/scripts/zulip-puppet-apply`.
 
 Daily full-database backups will be taken at 0200 UTC, and every WAL
-log will be written to S3 as it is saved by PostgreSQL; by default,
-this happens every 5 minutes, or every 1G, whichever happens first.
+archive will be written to S3 as it is saved by PostgreSQL; these are
+written every 16k of WAL log, which means that if there are periods of
+slow activity, it may be minutes before the backup is saved into S3 --
+see [`archive_timeout`][archive-timout] to set an upper bound on this.
+Because these are written every 16k, this may result in a large number
+of continuous PutObject calls in S3.
```

Comment:
Maybe the frequency would be better discussed as "On an active Zulip server, this means the Zulip server will be continuously sending PutObject requests to S3."